manly.py

```diff
@@ -155,4 +149,12 @@ def main(command):
     if not len(args.command):
         print("manly: missing COMMAND\n" "Try 'manly --help' for more information.")
     else:
-        main(args.command)
+        title, output=main(args.command)
+        # ---------- WRITE OUTPUT ---------- #
```

Comment:
Can you remove this comment and all other that looks like it (in `main`)? They don't fit in anymore imo.

Reply:
Should I remove all the comments of the type ------- xxxx -------- in the entire file?

Reply:
Removed all comments of the form ---- xxxx ---- in `main` and this one. :)